### PR TITLE
Fix for current time: Globally replace Time.now and Time.zone.now with Time.current

### PR DESCRIPTION
**Pull request by sujiayi317**
##  Motivation and Context

As sugested by Prof. Liu:
Problem: Currently MarkUs uses a few different methods to access the current time: Time.now, Time.zone.now, Time.current.
Solution: Do a global search-and-replace for the first two to replace them with Time.current.

## Your Changes

Globally replace Time.now and Time.zone.now with Time.current

**Description**:

**Type of change** (select all that apply):

 
- [x] Refactoring (internal change to codebase, without changing functionality)

## Testing

Run `docker-compose run test rspec`, 3562 examples, 0 failures, 23 pending

## Questions and Comments (if applicable)

Markus/coverage/index.html is read-only, so no opration is performed for this file.
All other 42 files are updated with Time.current (as a replacement of Time.now or Time.zone.now).

Steps I have taken: Run `git diff` to compare the changes. No side effect.

## Checklist
- [x] I have performed a self-review of my own code.
- [ ] I have fixed any Hound bot comments. 
- [x] I have verified that the TravisCI tests have passed. 
- [ ] I have reviewed the test coverage changes reported on Coveralls.

### CODE REVIEW

File: app/lib/git_repository.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ def self.do_commit(repo, author, message)
     index = repo.index
     commit_tree = index.write_tree(repo)
     index.write
-    commit_author = {email: 'markus@markus.com', name: author, time: Time.now}
+    commit_author = { email: 'markus@markus.com', name: author, time: Time.current }
```
Comment (david-yz-liu) [inline app/lib/git_repository.rb:65]: Looks like this change might be causing an error in the git tests (see Travis report). Please change back for now, we can look into this later.

Comment (sujiayi317) [reply app/lib/git_repository.rb:65]: Oh, yes! Sorry about that, I thought this change would be fine... Let me change back now.

Comment (david-yz-liu) [reply app/lib/git_repository.rb]: Oh, no the issue isn't the spaces, it's the `Time.now` vs. `Time.current` (I think). You should change this back to `Time.now`.

Comment (sujiayi317) [reply app/lib/git_repository.rb]: I've just changed this back to `Time.now`, hopefully this time it will work.

Comment (sujiayi317) [reply app/lib/git_repository.rb]: All checks have passed according to Travis, yeah. I'm a little confused why`Time.now` is the only correct version in this case.

**Pull request closed without merging**